controllers/User.controller.js

```diff
@@ -0,0 +1,86 @@
+const userDb = require("../models/user.model");
+
+module.exports.signUpUser = async (req, res) => {
+  const { name, email, password } = req.body;
+  let user = null;
+  try {
+    if (await userDb.findOne({ email: email })) {
```

Comment:
Here we can refactor:
`const user = await userDb.findOne({email:email})` 
if(user){....}